kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/monitor/ModemMonitorServiceImpl.java

```diff
@@ -168,6 +168,9 @@ protected void activate(ComponentContext componentContext) {
 
         stopThread = new AtomicBoolean();
 
+        // must be initialized before trackModem() is called; risk of NPE otherwise
+        this.m_executor = Executors.newSingleThreadExecutor();
```

Comment:
If it does not go ever 1k lines, I believe we can also rename the variables according to the new conventions.